zerver/lib/rest.py

```diff
@@ -98,74 +90,107 @@ def rest_dispatch(request: HttpRequest, **kwargs: Any) -> HttpResponse:
             target_function, view_flags = entry
         else:
             target_function = supported_methods[method_to_use]
```

Comment:
Could `return` immediately here, removing the `target_function` and `view_flags` variables.